Policies/Kubernetes/selinux/azurepolicy.json

```diff
@@ -0,0 +1,87 @@
+{
+  "properties": {
+    "displayName": "Ensure only allowed SELinux options are used in a kubernetes cluster",
+    "policyType": "Custom",
+    "mode": "Microsoft.Kubernetes.Data",
+    "description": "This policy ensures only allowed SELinux options are used in a Kubernetes cluster. For instructions on using this policy, please visit https://aka.ms/kubepolicydoc.",
+    "metadata": {
+      "version": "1.0.0-preview",
+      "category": "Kubernetes PSP",
+      "preview": true
+    },
+    "parameters": {
+      "effect": {
+        "type": "String",
+        "metadata": {
+          "displayName": "Effect",
+          "description": "Enable or disable the execution of the policy"
+        },
+        "allowedValues": [
+          "audit",
+          "deny"
+        ],
+        "defaultValue": "deny"
+      },
+      "excludedNamespaces": {
+        "type": "Array",
+        "metadata": {
+          "displayName": "Namespace exclusions",
+          "description": "List of Kubernetes namespaces to exclude from policy evaluation. Providing a value for this parameter is optional."
+        },
+        "defaultValue": []
+      },
+      "allowedSELinuxLevel": {
```

Comment:
Looking at REGO tests, I understand not all SELinux parameters have to be provided. But I see the SELinux parameters defined in Azure policy do not have default value which means all the parameter values are required during assignment. Can you please double check the below rego test scenario through Azure Policy?

test_input_seLinux_options_allowed_in_list_subset {
    input := { "review": input_review, "parameters": input_parameters_in_list_subset}
    results := violation with input as input
    count(results) == 0
}

Specifying only subset of properties is a valid scenario and we need to support it through Azure Policy. For example, look at a sample PSP shared by a customer in https://github.com/Azure/azure-arc-kubernetes-preview/issues/100. You can see only 'rule' property defined in SELinux.